cs.html

```diff
@@ -961,689 +1180,684 @@ <h2>Constructors</h2>
-                  <tr>
-                    <th>Parameter</th>
-                    <th>Type</th>
-                    <th>Nullable</th>
-                    <th>Optional</th>
-                    <th>Description</th>
-                  </tr>
-                  <tr>
-                    <td class="prmName">data</td>
-                    <td class="prmType"><code>Uint8Array</code></td>
-                    <td class="prmNullFalse"><span role="img" aria-label=
-                    "False">&#10008;</span></td>
-                    <td class="prmOptFalse"><span role="img" aria-label=
-                    "False">&#10008;</span></td>
-                    <td class="prmDesc"></td>
-                  </tr>
-                </tbody>
-              </table>
-              <div>
-                <em>Return type:</em> <code><a>QuicStreamReadResult</a></code>
-              </div>
-            </dd>
-            <dt><dfn><code>abortReading</code></dfn></dt>
-            <dd>
-              <p>A hard shutdown of the <code><a>QuicReadableStream</a></code>. It may be called
-              regardless of whether the <code><a>QuicReadableStream</a></code> object
-              was created by the local or remote peer. When the <code>abortReading()</code>
-              method is called, the user agent <em class="rfc2119" title="MUST">MUST</em>
-              run the following steps:</p>
-              <ol>
-                <li>Let <var>stream</var> be the <code><a>QuicReadableStream</a></code> object
-                which is about to abort reading.</li>
-                <li>If <var>stream</var>'s <a>[[\Readable]]</a> slot is <code>false</code>,
-                <a>throw</a> an <code>InvalidStateError</code> and abort these steps.</li>
-                <li>Set <var>stream</var>'s <a>[[\Readable]]</a> slot to <code>false</code>.</li>
-                <li>Clear the <var>stream</var>'s read buffer.</li>
-                <li>Let <var>transport</var> be the <code><a>QuicTransportBase</a></code>,
-                which the <var>stream</var> was created from.
-                <li>Remove the <var>stream</var> from the <var>transport</var>'s
-                <a>[[\QuicTransportReadableStreams]]</a> internal slot.</li>
-                <li>Let <var>abortInfo</var> be the first argument.</li>
-                <li>Start the closing procedure by sending a STOP_SENDING frame with its error
+              <li>Else, set <var>result</var>'s <code>finished</code> to false.</li>
+              <li>Return <var>result</var>.
+            </ol>
+            <table class="parameters">
+              <tbody>
+                <tr>
+                  <th>Parameter</th>
+                  <th>Type</th>
+                  <th>Nullable</th>
+                  <th>Optional</th>
+                  <th>Description</th>
+                </tr>
+                <tr>
+                  <td class="prmName">data</td>
+                  <td class="prmType"><code>Uint8Array</code></td>
```

Comment:
I'm not entirely familiar with JavaScript binary array type system, so just wanted to check: is it possible, with current method signature, to write at an arbitrary offset in some array? I.e. can I pass a view instead of the entire array?

Reply:
Yes, that's what a Uint8Array is: it's basically a view of an ArrayBuffer.